tests/t_signals.c

```diff
@@ -0,0 +1,83 @@
+#include <gdk-pixbuf/gdk-pixbuf.h>
+
+gint area_prepared_cb_count = 0;
```

Comment:
Please decleare a struct with these members, create a static instance in the main function and pass it as data to the callbacks.